### ISSUE TITLE
`cargo geiger --help` may need some work

### ISSUE THREAD

**Issue by vi** [anderejd/cargo-geiger#18]
```
$ cargo geiger --help
cargo-geiger 0.3.0
anderejd <rajder@gmail.com>
Display a tree visualization of a dependency graph
...
```

The description should be updated.

Also where is the option advertised on Reddit to filter out unused unsafes?

**Comment by anderejd** [comment]
Thanks for the report, that looks rather sloppy :D

Description fixed by: https://github.com/anderejd/cargo-geiger/commit/99382eb80d6924d6c5c1c6966f28d81bdfb392cc

> Also where is the option advertised on Reddit to filter out unused unsafes?

This functionality is always on in 0.3.0, sorry for the confusion. Every metric is now printed with two numbers divided by a slash, eg. 123/124, that means that 123 samples of that unsafe metric was used by the build and there are 124 samples total in that crate. Does that make sense?

**Comment by anderejd** [comment]
> Reddit post called that feature experimental, so I expected an opt-in.

Thanks for this feedback, communication is hard :) I just edited the reddit post to clarify that I consider the entire tool as an experiment, not just the new feature.

> And I see 2/2 where I expect 0/2 (for my own crate where I know those methods are not used).

Can you share this crate?

There are known issues with false positives (I suppose I should add that to the Known issues list). For example test functions are still included unless they are located in an `.rs` file that only contains test code, the used / not used filtering is done on `.rs` file level at this time.

> And openssl-sys is green and is used. (How can a -sys crate be green at all?)

This is unexpected, I just did a quick eyeball based check and it contains plenty of unsafe, as expected. I tried `cargo-geiger` on `cargo-tree` that I knew depend on openssl-sys and it gave me the numbers:

30/43      24/50        0/0    0/0     0/0

Are you getting 0/0 for all metrics?

There are at this time known issues / limitations for macro and build.rs generated unsafe code. See Known Issues in the README.md. https://github.com/anderejd/cargo-geiger#known-issues

Related to: https://github.com/anderejd/cargo-geiger/issues/7

**Comment by anderejd** [comment]
Closing this since the structopt description is fixed in master and the rest is addressed in issue #22
